File: Pod/Classes/RoundedView.m
//
//  RoundedImageView.m
//  xoints
//
//  Created by Logan Sease on 7/16/15.
//  Copyright (c) 2015 Logan Sease. All rights reserved.
//

#import "RoundedView.h"
#import "UIView+Rounding.h"

@implementation RoundedView


- (void)baseInit {
    [self round];
}

- (id)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        [self baseInit];
    }
    return self;
}

- (id)initWithCoder:(NSCoder *)aDecoder {
    if ((self = [super initWithCoder:aDecoder])) {
        [self baseInit];
    }
    return self;
}


@end

File: Pod/Classes/RoundedView.swift
//
//  RoundedView.swift
//  BlockStackBrowser
//
//  Created by lsease on 7/24/17.
//  Copyright © 2017 blockstack. All rights reserved.
//

import UIKit

public class RoundedView : UIView
{
    required public init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        customSetup()
    }
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        customSetup()
    }
    
    func customSetup()
    {

        self.backgroundColor = UIColor(red: 1, green: 1, blue: 1, alpha: 0.8)
        self.layer.borderColor = UIColor.lightGray.cgColor
        self.layer.borderWidth = 1
        self.layer.cornerRadius = 6
    }
}
